tlslite/messages.py

```diff
@@ -1765,3 +1759,61 @@ def parse(self, p):
 
     def write(self):
         return self.bytes
+
+
+class Heartbeat(object):
+    """
+    Handling Heartbeat extension from RFC 6520
+
+    @type message_type: int
+    @ivar message_type: type of message (response or request)
+
+    @type payload_length: int
+    @ivar payload_length: length of sending payload
+
+    @type payload: bytes
+    @ivar payload: payload
+    """
+    def __init__(self):
+        self.contentType = ContentType.heartbeat
+        self.message_type = 0
+        self.payload = bytearray(0)
+        self.padding = bytearray(0)
+
+    def create(self, message_type, payload, padding_length):
+        """Create heartbeat request or response with selected parameters"""
+        self.message_type = message_type
+        self.payload = payload
+        if not(15 < padding_length < 2**14 - len(self.payload) - 3):
+            raise ValueError("too short/long padding")
+        self.padding = getRandomBytes(padding_length)
+        return self
+
+    def createResponse(self):
+        """Creates heartbeat response based on request."""
+        heartbeat_response = Heartbeat().create(
+                                        HeartbeatExtensionMessages.response,
```

Comment:
How it should look like?
`
heartbeat_response = Heartbeat().create(
    HeartbeatExtensionMessages.response,
    self.payload,
    16)
`
is ok?